src/get-started/install.md

```diff
@@ -56,9 +56,12 @@ these packages from third-party PPAs, or build them from sources.
 
 ### Windows
 
-Exonum is a little behind supporting Windows OS compared to Linux and MacOS.
-In many Windows setups, Exonum will work just fine, but there are known problems
-with its compilation in certain configurations. If you have encountered a problem
+Install the latest version of the following packages:
+
+- [Visual C++ Build Tools][build_tools]
+- [PowerShell][powershell]
+
+If you have encountered a problem
```

Comment:
Isn't this redundant if we _do_ support Windows properly? We probably have references to Gitter/Github issues elsewhere, don't we?

Reply:
I don't have a strong opinion here, but I tend to agree. On the other hand, we definitely don't support (test) Windows platform equally to OS X/Linux, so this can make sense.

Reply:
I incline to agree with @dmitry-timofeev; this paragraph seems redundant.

Reply:
@slowli, do you mean paragraph "If you have encountered a problem..."?

Reply:
Yeah.